.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+jobs:
+  build:
+    docker:
+    - image: docker:18.09.1
+    environment:
+      IMAGE_NAME: gcr.io/opensourcecoin/ipfs-test-network
+    steps:
+    - setup_remote_docker
+    - checkout
+    - run:
+        name: Build image
+        command: |
+          docker build . --tag "$IMAGE_NAME"
+    - run:
+        name: Push image
+        command: |
+          if [[ "$CIRCLE_BRANCH" = "master" ]]; then
+            docker login -u _json_key -p "$(echo $GCLOUD_SERVICE_KEY | base64 -d)" https://gcr.io
```

Comment:
Can we document somewhere to which account this key belongs?

Reply:
Done